music_assistant/server/providers/ytmusic/__init__.py

```diff
@@ -650,11 +633,8 @@ async def _parse_artist(self, artist_obj: dict) -> Artist:
 
     async def _parse_playlist(self, playlist_obj: dict) -> Playlist:
         """Parse a YT Playlist response to a Playlist object."""
-        playlist_id = (
-            f"{YT_YOUR_LIKES_PLAYLIST_ID}-{self.instance_id}"
-            if playlist_obj["id"] == YT_YOUR_LIKES_PLAYLIST_ID
-            else playlist_obj["id"]
-        )
+        # Playlist ID's are not unique across instances for lists like 'Likes', 'Supermix', etc.
+        playlist_id = f"{playlist_obj['id']}{YT_PLAYLIST_ID_DELIMITER}{self.instance_id}"
         playlist = Playlist(item_id=playlist_id, provider=self.domain, name=playlist_obj["title"])
```

Comment:
```suggestion
        playlist = Playlist(item_id=playlist_id, provider=self.instance_id, name=playlist_obj["title"])
```

I believe this is a somewhat easier solution to the problem. Set provider to the instance id for all playlists